Tensile/cmake/TensileConfig.cmake

```diff
@@ -178,6 +179,10 @@ function(TensileCreateLibraryFiles
     set(Options ${Options} "--cxx-compiler=${Tensile_COMPILER}")
   endif()
 
+  if(Tensile_COMPILER_PATH)
+    set(Options ${Options} "--cmake-cxx-compiler=${Tensile_COMPILER_PATH}")
+  endif()
+
```

Comment:
Why is this not just included in the --cxx-compiler above?

Reply:
--cxx-compiler is used to switch between hipcc and hcc and should probably be removed at this point. Mainly added a new path option to avoid problems where --cxx-compiler option does not support having a full path, just a compiler name to switch on